arch/unitroot/unitroot.py

```diff
@@ -1714,3 +1730,63 @@ def kpss_crit(stat, trend='c'):
     crit_value = interp(cv, y[::-1], x[::-1])
 
     return pvalue, crit_value
+
+def BandWidthNW(y, kernel = "ba"):
```

Comment:
I was referring to this function.

Reply:
I did not intend to do anything new on the Bandwidth, sorry.